src/site/_filters/get-paths.js

```diff
@@ -14,7 +14,24 @@
  * limitations under the License.
  */
 const postToPathsMap = require('../_data/postToPaths.js');
+const paths = require('../_data/paths');
 
 module.exports = (post) => {
-  return postToPathsMap[post.fileSlug];
+  const out = postToPathsMap[post.fileSlug];
+  if (out) {
+    return out;
+  }
+
+  // Not all posts are correctly attributed to a path (they're not linked to in the path JSON).
+  // However, guess based on their filename. This is only needed for some "return" links.
+  const parts = post.filePathStem.split('/');
```

Comment:
This seems good but worth mentioning that as part of the i18n work we were thinking of getting rid of directories like /fast/, /accessible/, etc. Though it's unclear when we would actually do that.